src/flux/course.coffee

```diff
@@ -46,7 +47,7 @@ CourseConfig =
     @emit('practice.loaded', obj.id)
 
   _reset: ->
-    CrudConfig.reset.call(@)
+    CrudConfig.reset.call(this)
```

Comment:
While it *does* find code like `that = @` I agree it's awkward. I'll probably just remove `no_stand_alone_at` from `coffeelint.json`